x-pack/metricbeat/module/mssql/_meta/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/mssql-server-linux:2017-latest
```

Comment:
Nice that we have a docker image to test this too 👍

Reply:
I'd like to work in this module on Windows too, setting up tests and versions to make it really reliable but I'd prefer to do it in a follow up PR.

Reply:
Yes, leave windows support for a future change, I don't think we have Windows CI machines with native docker.